src/consensus/pbft/pbft.h

```diff
@@ -195,19 +212,27 @@ namespace pbft
       message_receiver_base->register_reply_handler(
         reply_handler_cb, client_proxy.get());
 
-      auto global_commit_cb = [](kv::Version version, void* ctx) {
+      auto global_commit_cb = [](kv::Version version, ::View view, void* ctx) {
```

Comment:
Fix issue of pbft global namespace.

Reply:
https://github.com/microsoft/CCF/issues/585